sass/layouts/_mainContainer.scss

```diff
@@ -1,2 +1,4 @@
 .mainContainer {
+  display: flex;
+  flex-direction: column;
 }
```

Comment:
Nie powinnyśmy chyba tego commitować. Moga powstać konflikty u kogoś, kto będzie nad tym plikiem/kalsą pracował. Ty dodawałaś Tylko przycisk, a nie zmieniałaś inne komponenty

Reply:
Jeżeli kilka osób pracuje na tym samym pliku, to niestety nie unikniemy konfliktów. Ja robię, jako moje pierwsze zadanie, component, który powinien być ostatnim elementem w mainCointainer, chciałam go chociaż jako tako ustawić. Style obejmują dwie proste linijki kodu i myślę, że to nie będzie jakiś duży problem, aby to ponownie poustawiać ;)

Reply:
Jasne, dlatego warto u siebie to oscylować i pozmieniać, ale już nie commitować zmian w tym pliku.
Czyli zmieniasz ten plik i nie commitujesz i nie wypychasz tych niepowołanych zmian na serwer